operator_ui/src/components/Jobs/parseDot.ts

```diff
@@ -0,0 +1,25 @@
+import graphlibDot from 'graphlib-dot'
+
+export type Stratify = {
+  id: string
+  parentIds: string[]
+  attributes: { [key: string]: string }
```

Comment:
```suggestion
  attributes?: { [key: string]: string }
```
Would this make it easier to check whether there are any attributes?